threedi_modelchecker/config.py

```diff
@@ -1139,6 +1139,20 @@ def is_none_or_empty(col):
         ),
         message="v2_global_settings.interception_global is recommended as fallback value when using an interception_file.",
     ),
+    QueryCheck(
+        error_code=326,
+        level=CheckLevel.INFO,
+        column=models.SimpleInfiltration.id,
+        invalid=Query(models.SimpleInfiltration)
+        .join(
+            models.GlobalSetting,
+            models.SimpleInfiltration.id
+            == models.GlobalSetting.simple_infiltration_settings_id,
```

Comment:
This is now comparing against *all* global settings, while we only use the *first* global settings record. So you'll need to compare the `SimpleInfiltration.id` with a scalar subquery (defined above as `first_setting` or something like that)